.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,23 @@
+
+name: Automerge Dependabot
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  automerge:
+    name: Merge Dependabot pull request
```

Comment:
Maybe this is pedantic, but `Merge Dependabot Pull Request` has more consistent capitalisation